test/Microsoft.AspNet.Authentication.Test/OAuthBearer/OAuthBearerMiddlewareTests.cs

```diff
@@ -27,6 +26,8 @@ public class OAuthBearerMiddlewareTests
         {
             var server = CreateServer(options =>
             {
+                options.AutomaticAuthentication = true;
```

Comment:
Now that a PR has been submitted to reunify `AuthenticationHandler` and `AutomaticAuthenticationHandler`, I suggest moving this discussion there: https://github.com/aspnet/Security/pull/189